scripts/dev/zero-downtime-deploy.sh

```
@@ -0,0 +1,231 @@
+LOG_FILE="$APP_DIR/deploy.log"
+BLUE_PORT=8080
+GREEN_PORT=8081
+BLUE_MONITOR_PORT=8083
+GREEN_MONITOR_PORT=8084
+MAX_HEALTH_CHECK_RETRIES=60
+HEALTH_CHECK_INTERVAL=2
+PROFILE="dev"
+TIMEZONE="Asia/Seoul"
+
+log() {
+    local timestamp=$(date '+%Y-%m-%d %H:%M:%S')
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)